# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: target/archivista/pkg_linux26.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Disable SMP by default, it can be explicitly turned on again by adding maxcpu= ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.18/init/main.c.orig	2006-10-13 15:37:04.000000000 +0200
+++ linux-2.6.18/init/main.c	2006-10-13 15:38:35.000000000 +0200
@@ -125,7 +125,7 @@
 static char *ramdisk_execute_command;
 
 /* Setup configured maximum number of CPUs to activate */
-static unsigned int max_cpus = NR_CPUS;
+static unsigned int max_cpus = 0;
 
 /*
  * Setup routine for controlling SMP activation
